openmc/data/function.py

```diff
@@ -9,7 +11,49 @@
                         4: 'log-linear', 5: 'log-log'}
 
 
-class Tabulated1D(object):
+class Function1D(with_metaclass(ABCMeta, object)):
```

Comment:
If you're ok with it, I'd suggest holding off on using six for this. If we're going to adopt a dependency on `six` we might as well use it judiciously in other areas as well (type checking for example). I think the make html issues will still be fixed even without fixing the metaclass specification here.